src/main/java/fi/metatavu/soteapi/persistence/dao/ContentDAO.java

```diff
@@ -32,17 +33,22 @@
    * @param contentType content type
    * @param parent parent content
    * @param category category slug
+   * @param created created
+   * @param modified modified
    * @param orderIndex order index
+   * @param archived archived
    * @return created content
    */
-  public Content create(String origin, String originId, String slug, ContentType contentType, Content parent, String category, Long orderIndex, Boolean archived) {
+  public Content create(String origin, String originId, String slug, ContentType contentType, Content parent, String category, OffsetDateTime created, OffsetDateTime modified, Long orderIndex, Boolean archived) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 10 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)